notebooks/util.py

```diff
@@ -179,7 +179,7 @@ def cluster_molecules(fingerprints, cutoff=0.6):
         
     Returns
     -------
-    list of tuple of int
+    list of tuple of integers (ids)
```

Comment:
In numpy docstrings, this position is reserved for python datatypes (in this case `int`). In the next line, we can add the description for this `int`, i.e. that this integer refers to an ID of some sort.

```
Returns
-------
int
   Cluster IDs.
```